packages/nimble-components/src/table/template.ts

```diff
@@ -79,7 +79,7 @@ export const template = html<Table>`
                                 ${ref('collapseAllButton')}
                                 ${'' /* tabindex managed dynamically by KeyboardNavigationManager */}
                                 tabindex="-1"
-                                class="collapse-all-button ${x => `${x.showCollapseAll ? 'visible' : ''}`}"
+                                class="collapse-all-button ${x => `${x.showCollapseAll ? 'visible' : ''}`} ${x => `${x.canHaveCollapsibleRows ? '' : 'hidden'}`}"
```

Comment:
Would it be better to have a new volatile property that returned the right CSS setting:
```
@volatile
public get collapseButtonVisibility(): string {
    if (!this.canHaveCollapsibleRows) {
        return 'hidden';
    } else if (this.showCollapseAll) {
        return 'visible';
    }
    return '';
}
```
The thing that rubs me the wrong way is having two independent observables competing to set opposite CSS state.